3.14.1 on FreeBSD

# syslog-ng

## Version of syslog-ng

syslog-ng 3.14.1 release

## Platform

FreeBSD 11.1


# Issue

Compilation of syslog-ng on FreeBSD 11.1 fails with the following error:
```
cc -DHAVE_CONFIG_H -I.  -I./lib -I./modules -I./lib -I./modules -I/usr/local/include/glib-2.0 -I/usr/local/lib/glib-2.0/include -I/usr/local/include -pthread  -I./lib/eventlog/src -I./lib/eventlog/src -I/usr/local/include  -I/usr/include   -I./lib/ivykis/src/include -I./lib/ivykis/src/include -D_GNU_SOURCE -D_DEFAULT_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64  -O2 -pipe -fstack-protector -fno-strict-aliasing -Wall -pthread -MT syslog-ng-ctl/syslog-ng-ctl.o -MD -MP -MF $depbase.Tpo -c -o syslog-ng-ctl/syslog-ng-ctl.o syslog-ng-ctl/syslog-ng-ctl.c &&\
mv -f $depbase.Tpo $depbase.Po
syslog-ng-ctl/syslog-ng-ctl.c:35:10: fatal error: 'sgtty.h' file not found
#include <sgtty.h>
         ^~~~~~~~~
1 error generated.
gmake[4]: *** [Makefile:11708: syslog-ng-ctl/syslog-ng-ctl.o] Error 1
gmake[3]: *** [Makefile:17482: all-recursive] Error 1
gmake[2]: *** [Makefile:6791: all] Error 2
```

--- a/syslog-ng-ctl/syslog-ng-ctl.c
+++ b/syslog-ng-ctl/syslog-ng-ctl.c
@@ -32,7 +32,6 @@
 #include <string.h>
 #include <stdlib.h>
 #include <locale.h>
-#include <sgtty.h>
 #include <termios.h>
 #include <unistd.h>
 #include <errno.h>

